tests/test_cfg/test_label_if.py

```diff
@@ -68,3 +75,53 @@ def test_label_if_elsif() -> None:
     expected_false_labels = 3
     expected_true_labels = 3
     assert _extract_edge_labels(build_cfg(src)) == (expected_true_labels, expected_false_labels)
+
+
+def test_condition_if_no_else() -> None:
+    """Test that the condition nodes from the if condition were correctly updated for an if block with no
+    else statement."""
+    src = """
+    x = 0
+    if x > 0:
+        x = 4
+    """
+    expected_num_conditions = 2
+    found_conditions = _extract_edge_conditions(build_cfg(src))
+    assert all(isinstance(condition, astroid.nodes.Compare) for condition in found_conditions)
```

Comment:
I'd like to see more specificity here. You can use the `as_string` method (or something similar) to convert a `NodeNG` into a string to actually check that the correct condition is stored; same with all of the tests that you've added.